src/code/z_camera.c

```diff
@@ -4133,212 +4113,204 @@ s32 Camera_KeepOn1(Camera* camera) {
         Camera_ScaledStepToCeilF(1.0f, camera->yOffsetUpdateRate, camera->speedRatio * 0.5f, 0.0001f);
     camera->xzOffsetUpdateRate =
         Camera_ScaledStepToCeilF(1.0f, camera->xzOffsetUpdateRate, camera->speedRatio * 0.2f, 0.0001f);
-    // TODO: No f on 0.05?
-    camera->fovUpdateRate = Camera_ScaledStepToCeilF(0.05, camera->fovUpdateRate, camera->speedRatio * 0.05f, 0.0001f);
+
+    camera->fovUpdateRate = Camera_ScaledStepToCeilF(.05f, camera->fovUpdateRate, camera->speedRatio * 0.05f, 0.0001f);
```

Comment:
ouch

Reply:
This can actually be explain by OoT:
https://github.com/zeldaret/oot/blob/a20163c6abf7700c432c711b03532cc4ad8d198a/src/code/z_camera.c#L3226-L3227

this float was a REG in OoT and left to a hard constant in MM, so it's probably a remnant of that. Might even be `5 * 0.01f` because of the `CAM_DATA_SCALED`?

Reply:
I think it is worth a shot trying the `5 * 0.01f`
Btw why does MM doesn't have `CAM_DATA_SCALED`?

Reply:
In OoT, it's currently called
`CAM_DATA_SCALED`
But in MM, there's both
```c
#define CAM_RODATA_SCALE(x) ((x)*100.0f)
#define CAM_RODATA_UNSCALE(x) ((x)*0.01f)
```
tbh it's been so long since I looked at this, I don't remember which came first and which came second. But they'll be synced eventually

And yeah, doesn't match